<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Necromouser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbSerialPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port</string>
    </property>
    <widget class="QComboBox" name="cbSerialPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select the serial port your pointing device is connected to with this dropdown.
Changing this will require clicking start again.</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbProtocol">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Protocol</string>
    </property>
    <widget class="QComboBox" name="cbProtocal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select the protocal your pointing device uses with this dropdown.
Changing this will require clicking start again.</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="ckbMiddleClickEmulation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Emulates a middle click by pressing the left and right mouse buttons at the same time.
If Emulate Scrolling is enabled, middle click will only be emulated if the pointer does not move while the buttons are down and the pressed and released events will occure when both buttons are released.
Changing this will have an immediate effect and does not require pressing Start again.</string>
    </property>
    <property name="text">
     <string>Emulate Middle Click (Left + Right)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbScrollEmulation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Emulates scrolling by middle clicking (native or emulated) and moving the mouse before letting go of the click.
With this enabled, middle clicks will not occure unless the mouse is not moved and both the pressed and released events will occure when the middle button is released.
Changing this will have an immediate effect and does not require pressing Start again.</string>
    </property>
    <property name="text">
     <string>Emulate Scrolling (Middle + Mouse movement)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbScrollSensitivity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Scroll Sensitivity Divisor</string>
    </property>
    <widget class="QSlider" name="hsScrollSensitivity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Changes the sensitivity of the scrolling emulation.
Lower values are more sensitive. </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblSensitivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Starts or restarts Necromouser on the specified port and with the specified protocal.</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Started...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbSwapAxis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>241</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Swaps the X and Y axis..
Changing this will have an immediate effect and does not require pressing Start again.</string>
    </property>
    <property name="text">
     <string>Swap Axis</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbAcceleration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Acceleration</string>
    </property>
    <widget class="QLabel" name="lblXAccel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYAccel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbXAccel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>43</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Multiplies the movement in the X direction.
Changing this will have an immediate effect and does not require pressing Start again.</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbYAccel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>43</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Multiplies the movement in the Y direction.
Changing this will have an immediate effect and does not require pressing Start again.</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbInversion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Inversion</string>
    </property>
    <widget class="QCheckBox" name="ckbInvertX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Inverts the X axis.
Changing this will have an immediate effect and does not require pressing Start again.</string>
     </property>
     <property name="text">
      <string>Invert X</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbInvertY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Inverts the Y axis.
Changing this will have an immediate effect and does not require pressing Start again.</string>
     </property>
     <property name="text">
      <string>Invert Y</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="ckbDebugMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>241</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Necromouser will only read data from the serial port and generate packets.
It will not parse or otherwise attempt to use those packets.</string>
    </property>
    <property name="text">
     <string>Protocol Reader Debug Mode</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
